templates/MoneyPolitics/PreparingMessage.html

```diff
@@ -159,7 +159,7 @@
       {% endblocktrans %}
     </p>
     <p>
-      {% blocktrans with message_cost=Constants.message_cost %}
+      {% blocktrans %}
```

Comment:
Because you are in a {% blocktrans %}, you have to define the variables you're going to use inside the block. If not, things kinda go crazy with the translation.

To fix this, you can change `{% blocktrans %}` to `{% blocktrans with message_cost=message_cost %}`. Check first if that works. If not, I'll check for another way.

- In order to edit the code for a Pull Request, you only need to make a new push to the compare branch (in this case, yours)